processor/ratelimitprocessor/config.go

```diff
@@ -102,19 +134,82 @@ type GubernatorBehavior string
 
 func createDefaultConfig() component.Config {
 	return &Config{
-		Strategy:         StrategyRateLimitRequests,
-		ThrottleBehavior: ThrottleBehaviorError,
-		Type:             LocalRateLimiter,
+		Type: LocalRateLimiter,
+		RateLimitSettings: RateLimitSettings{
+			Strategy:         StrategyRateLimitRequests,
+			ThrottleBehavior: ThrottleBehaviorError,
+			ThrottleInterval: 1 * time.Second,
```

Comment:
nit: Define it as constant as in here https://github.com/elastic/hosted-otel-collector/blob/accbdc7cacb29450ca61e0c052a4da5bbf624ca2/internal/extension/elasticcloudauthextension/config.go#L46

Just to have same style